mab/local_game/local_entities/local_map_features/local_feature.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC
+
+from local_entities.local_entity import LocalEntity, LocalEntities
+from local_map.local_hex import LocalHex
+
+
+class LocalFeature(LocalEntity, ABC):
```

Comment:
why ABC?))

Reply:
This is probably a left over from a previous commit, will delete